app/gameengine.py

```diff
@@ -3152,6 +3193,9 @@ def archive_hand_continuation():
                         case "event":
                             # Only include cards that are events.
                             cards_can_choose = [card for card in cards_can_choose if is_card_event(card)]
+                        case "support_types":
```

Comment:
feels like we should make the other cases here also use this instead of having individual per sub_type.

Reply:
This would be best but I would like for it not to be part of this PR. But would be better to do it in a separate ``bug_fix/improvement`` PR so that only relevant changes are included here.